### PR TITLE
feat: push css with http2

### PR DESCRIPTION
**Pull request by tilmx**
for faster pageloads

### CODE REVIEW

File: lib/_headers [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/*
+  Link: </css/style.css>; rel=preload; as=stylesheet
```
Comment (markusoelhafen) [inline lib/_headers]: Add a newline at the end of code

Comment (tilmx) [reply lib/_headers]: Why?

Comment (markusoelhafen) [reply lib/_headers]: Sorry, there should have been an explanation :)
Newlines on the end of a file is not only a common coding style, but can lead to some issues.
[stackoverflow](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Comment (tilmx) [reply lib/_headers]: done!

**Comment by tilmx** [comment]
we should squash the commits...

**Review by markusoelhafen — APPROVED** [review]

**Pull request merged**